spec/models/manageiq/providers/kubernetes/container_manager/scanning/job_spec.rb

```diff
@@ -144,6 +144,20 @@ def fetch_metadata(*_args)
       end
     end
 
+    it 'should add correct environment variables' do
+      att_name = 'http_proxy'
+      my_value = "MY_TEST_VALUE"
+      @ems.custom_attributes.create(:section => described_class::ATTRIBUTE_SECTION,
+                                    :name    => att_name,
+                                    :value   => my_value)
+      allow_any_instance_of(described_class).to receive_messages(:kubernetes_client => MockKubeClient.new)
```

Comment:
@jrafanie I can't remember if `allow_any_instance_of` is the bad one to use, but can you comment here?